src/SwedbankPay.Sdk/PaymentInstruments/Card/CardPaymentAbortRequest.cs

```diff
@@ -1,23 +1,23 @@
-﻿namespace SwedbankPay.Sdk.Payments.CardPayments
-{
-    public class CardPaymentAbortRequest
-    {
-        public CardPaymentAbortRequest(string abortReason)
-        {
-            this.Payment = new CardPaymentAbortPayment(abortReason);
-        }
-        public CardPaymentAbortPayment Payment { get; }
-
-        public class CardPaymentAbortPayment
-        {
-            public string Operation = "Abort";
-
-            public CardPaymentAbortPayment(string abortReason)
-            {
-                AbortReason = abortReason;
-            }
-
-            public string AbortReason { get; set; }
-        }
-    }
+﻿namespace SwedbankPay.Sdk.PaymentInstruments.Card
+{
+    public class CardPaymentAbortRequest
+    {
+        public CardPaymentAbortRequest(string abortReason)
+        {
+            this.Payment = new CardPaymentAbortPayment(abortReason);
+        }
+        public CardPaymentAbortPayment Payment { get; }
+
+        public class CardPaymentAbortPayment
+        {
+            public string Operation = "Abort";

```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)